WcaOnRails/spec/models/competition_wcif_spec.rb

```diff
@@ -18,6 +18,8 @@
       event_ids: %w(333 444 333fm 333mbf),
       with_schedule: true,
       competitor_limit: 50,
+      registration_open: "2013-12-01",
+      registration_close: "2013-12-31",
```

Comment:
Without this, I was getting a validation error, since the `registration_open`/`registration_close` are [54 weeks ago](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/spec/factories/competitions.rb#L99-L100), whereas the `start_date` earlier in this file is 2014.